include/tvm/script/ir_builder/tir/frame.h

```diff
@@ -186,17 +194,33 @@ class AssertFrameNode : public TIRFrameNode {
   TVM_DECLARE_FINAL_OBJECT_INFO(AssertFrameNode, TIRFrameNode);
 
  public:
+  /*!
+   * \brief The method called when exiting RAII scope.
+   * \sa tvm::support::With
+   */
   void ExitWithScope() final;
 };
 
+/*!
+ * \brief Reference type of AssertFrameNode.
```

Comment:
It's somewhat a convention in TVM to say so. e.g.: https://github.com/apache/tvm/blob/main/include/tvm/meta_schedule/search_strategy.h#L186

```suggestion
 * \brief Managed reference to AssertFrameNode.
```